pkg/network/node/iptables.go

```diff
@@ -27,6 +28,23 @@ type NodeIPTables struct {
 	egressIPs map[string]string
 }
 
+var iptablesBackoff wait.Backoff = wait.Backoff{
+	Duration: 500 * time.Millisecond,
+	Factor:   1.25,
+	Steps:    10,
+}
```

Comment:
assuming I did the math right, this will wait a total of about 13 seconds. Worth a comment

Reply:
done